### PR TITLE
Search author name

### PR DESCRIPTION
**Pull request by yemoski**

### CODE REVIEW

**Comment by nein09** [comment]
@yemoski If you push up what you have in this branch (with the resolved merge conflict) it'll update this pull request and I can take a look. Thanks!

**Comment by yemoski** [comment]
I just resolved the merge conflict, let me know if the tests I wrote look good.
Thanks!

**Comment by nein09** [comment]
Ahh, sorry - my other PR caused another conflict. I should have thought about that before hitting that merge button.

**Comment by yemoski** [comment]
oh sorry!
how does it look now?

File: app/search/gleaner.py [hunk 1]
```diff
@@ -189,7 +200,9 @@ def execute_query(self, page_number):
 
     def text_search(self, **kwargs):
         text = kwargs.pop('text', None)
+        author = kwargs.pop('author',None)
         page_number = kwargs.pop('page_number', 0)
+        print('author gleaner')
```
Comment (nein09) [inline app/search/gleaner.py:205]: Please remove this debug print statement

**Review by nein09 — COMMENTED** [review]
This looks good except for the one stray print statement. What I want to do is get my new computer up and running and then try this out on there - if it works well, I'll merge it in. Thank you!

**Comment by nein09** [comment]
I just merged main in and had a look....

and I got 

```Response:
b'MALFORMED QUERY: Encountered " "contains" "CONTAINS "" at line 68, column 17.
```

I think you might need `FILTER CONTAINS` instead of just plain `CONTAINS` - it's worth doing some experimenting with the sparql workbench to see whether the queries that gleaner.py generates are acceptable syntax. And, of course, if you manage to get your triplestore going, you can see if it finds the authors you want!

**Comment by yemoski** [comment]
oh, i see, I would try writing a couple of things to the triple store to see if I get any results.

**Comment by nein09** [comment]
That's a good idea, or if you let the write to triplestore go overnight, you would hopefully have the whole data set.

**Pull request merged**